client/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,162 @@
+import { AppBar, Toolbar, Typography, Badge, IconButton, Drawer, Box, Grid } from '@material-ui/core';
+
+import { Link } from 'react-router-dom';
+import MenuIcon from '@material-ui/icons/Menu';
+import React, { useState, useEffect } from 'react';
+import AvatarDisplay from '../AvatarDisplay/AvatarDisplay';
+import useStyles from './useStyles';
+import PetsIcon from '@material-ui/icons/Pets';
+import { User } from '../../interface/User';
+
+interface Props {
+  logOut: string;
+  profile: string;
+  mySitter: string;
+  becomeSitter: string;
+  messages: string;
+  loggedIn: boolean;
+  loggedInUser: User;
+  open: boolean;
+}
+
+const Navbar = ({
+  logOut,
+  profile,
+  mySitter,
+  becomeSitter,
+  messages,
+  loggedInUser,
+  loggedIn,
+  open,
+}: Props): JSX.Element => {
+  const classes = useStyles();
+
+  const [state, setState] = useState({
+    mobileView: false,
+    drawerOpen: false,
+  });
```

Comment:
Especially with functional components & hooks, we should try to use a single `useState` hook for each part of the state (e.g. one for `mobileView` and one for `drawerOpen`) unless we have a good reason not to. I think these would definitely be better separated.

Reply:
created two separate states